incubator/elasticsearch-curator/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "5.4.1"
 description: A Helm chart for Elasticseach Curator
 name: elasticsearch-curator
-version: 0.1.1
+version: 0.1.2
```

Comment:
you are introducing a breaking change, please bump to 0.2.1

Reply:
@gianrubio shouldn't it be `0.2.0` instead of `0.2.1`